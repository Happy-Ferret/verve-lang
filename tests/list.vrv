print([1, 2, 3])
print(["foo", "bar"])
print(head([3, 2, 1]))
print(tail([3, 2, 1]))
print(length([3, 2, 1, 4, 7]))
