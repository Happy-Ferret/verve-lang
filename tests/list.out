1 2 3
foo bar
3
2 1
5
